dart/test/flx_reader_test.dart

```diff
@@ -0,0 +1,647 @@
+import 'dart:typed_data';
```

Comment:
Test files should sit under `tests` ideally, not `dart`

Reply:
I just used already given infrastructure. There were already test files in `test` folder. It is also the default folder structure in Dart:
- lib, for sources
- test, for unit tests
- example, for sample usage 
This file structure is what Dart tooling expects and support in the best case.
However if it is very important I could move the files.